pkgs/development/compilers/bluespec/default.nix

```diff
@@ -18,20 +18,26 @@
 , gmp-static
 , verilog
 , asciidoctor
-, tex }:
+, tex
+, which }:
```

Comment:
```suggestion
, which
}:
```

Reply:
Thanks, applied OOB while fixing the hydra eval failure.